student.rb

```diff
@@ -0,0 +1,21 @@
+require './person'
+
+class Student < Person
+  attr_reader :classroom, :parent_permission
+
+  def initialize(age, name = 'unknown', parent_permission: true)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+    super
+    @classroom = classroom
```

Comment:
- [ ] Kindly use your super to send your parameters to the parent class as shown below 👍 
 
⬇️
`super(@age, @name, @parent_permission)`